src/api/stats.ts

```diff
@@ -143,13 +145,15 @@ const getStatsByName = (names: string[], filter?: StatsFilter) => {
         // Month Range
         case 'thisMonth':
             queryBuilder = queryBuilder
-                .eq('ts', formatToGMT(startOfMonth(today)))
+                .gte('ts', formatToGMT(startOfMonth(today)))
```

Comment:
this is kinda smelly, I don't know why a time range would be required.

For reference, here's how our reporting is mapping a `Date` into truncated time points: https://github.com/estuary/flow/blob/master/ops-catalog/catalog-stats.ts#L54-L79